subsystem/subsystem_suite_test.go

```diff
@@ -117,6 +120,14 @@ func init() {
 			logrus.Fatal("Failed to init wiremock stubs, ", err)
 		}
 	}
+
+	// Test on first openshift version. We can't test on latest because quay.io/openshift-release-dev/ocp-release-nightly
```

Comment:
Wouldn't it break the Kube API subsystem tests that require 4.8 version? @filanov

Reply:
kube-api subsystem install SNO and multiple hosts cluster, so it will need both 4.7 and 4.8

Reply:
4.8 can't work right now, because of the comment I wrote here. The repo `quay.io/openshift-release-dev/ocp-release-nightly` is not publicly available, so the image can't be pulled.

Reply:
4.8 can't work right now. Repo `quay.io/openshift-release-dev/ocp-release-nightly` is not publicly available, so the image can't be pulled with creds we use right now.